L/LibGit2/bundled/patches/libgit2-hostkey.patch

```diff
@@ -1,24 +1,5 @@
-diff --git a/include/git2/cert.h b/include/git2/cert.h
-index e8cd2d180..54293cd31 100644
---- a/include/git2/cert.h
-+++ b/include/git2/cert.h
-@@ -111,6 +111,14 @@ typedef struct {
- 	 * have the SHA-256 hash of the hostkey.
- 	 */
- 	unsigned char hash_sha256[32];
-+
-+	/**
-+	 * Hostkey itself.
-+	 */
-+	int hostkey_type;
-+	size_t hostkey_len;
-+	unsigned char hostkey[1024];
```

Comment:
Mmm, why did you remove this? It sounds like an essential part of the patch.

Reply:
Because it seems to be handled by upstream code now and looked redundant.

Reply:
Great, indeed I see it at https://github.com/libgit2/libgit2/blob/v1.2.0/include/git2/cert.h#L108-L151.